src/common/models/Projection.ts

```diff
@@ -464,7 +483,7 @@ export class Projection {
     }
     // Get last week of sane data (ignore negative values)
     const lastWeekOfPositives = _.filter(
-      this.dailyPositiveTests.slice(Math.max(0, i - 6), i + 1),
+      data.slice(Math.max(0, i - 6), i + 1),
```

Comment:
Maybe set `Math.max(i + 1, data.length)` just to make sure that the end on `slice` is also valid.

Reply:
Hrm.  Do you mean `Math.min(i+1, data.length)`?  But I think it's probably invalid for `i` to be `>= data.length`.  So I don't think that should be necessary.  If we wanted to be cautious, we could `assert (i < data.length, ...)` above somewhere.  But it doesn't seem super valuable, especially if we're going to get rid of this code soon and use smoothWithRollingAverage() 😁

Reply:
punting for now